corgi/core/models.py

```diff
@@ -1089,16 +1063,9 @@ def manifest_components(self, ofuri: str = "", quick=False) -> "ComponentQuerySe
         if not quick:
```

Comment:
I guess we should remove the 'quick' function parameter as it no longer has any affect.

Reply:
Want to avoid adding any more potential blast radius to this PR ... something for the (near) future.